tests/data/procurve_131.json

```diff
@@ -3,12 +3,12 @@
         "discovery": {
             "devices": [
                 {
-                    "sysName": "",
+                    "sysName": null,
                     "sysObjectID": ".1.3.6.1.4.1.11.2.3.7.11.131",
                     "sysDescr": "HP J9625A 2620-24-PoEP Switch, revision RA.16.02.0012, ROM RA.15.13 (/ws/swbuildm/rel_spokane_qaoff/code/build/xform(swbuildm_rel_spokane_qaoff_rel_spokane))",
                     "sysContact": null,
                     "version": "RA.16.02.0012",
-                    "hardware": "2620-24-PoEP",
+                    "hardware": "J9625A 2620-24-PoEP",
```

Comment:
I'm thinking we might want to filter out the JxxxxA part, pattern seems to be `J[A-Z0-9]{4}A` or maybe `J[A-Z0-9][0-9]{3}A`

Reply:
I specifically included the part number.  I find it more useful than the model name.

Reply:
I agree to @murrant. Would it make sense to put this information in the (actually empty) features tag?

Reply:
Not straight forward to do. You have this format: 516733-B21, some end with B, or C, sometimes it is after the model and sometimes before, and sometimes there is no model and only the part number.

Reply:
That's a point...

Reply:
@mpikzink What I mean is, you can't split out the part number without doubling or more the number of regexes required.